src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -53,6 +54,18 @@ class DataDownload extends AsyncView<Props, State> {
     return [['download', `/organizations/${orgId}/data-export/${dataExportId}/`]];
   }
 
+  getActionLink(queryType): string {
+    const {orgId} = this.props.params;
+    switch (queryType) {
+      case ExportQueryType.IssuesByTag:
+        return `/organizations/${orgId}/issues/`;
+      case ExportQueryType.Discover:
+        return `/organizations/${orgId}/discover/queries/`;
+      default:
+        return '/';
+    }
+  }
```

Comment:
This will display a CTA for the expired pages. In the future we could make this go directly to the issue for their expired download, but for now retention policies and 404s make this a little bit difficult.